pom.xml

```diff
@@ -82,6 +82,11 @@
             </exclusions>
         </dependency>
         <dependency>
+            <groupId>org.apache.ivy</groupId>
+            <artifactId>ivy</artifactId>
+            <version>2.4.0</version>
```

Comment:
The actual fix. Groovy includes Grape but `groovy-all` depends on the sole engine, based on Ivy, which is not otherwise in the classpath. Seems to suffice for it to be somewhere in the `uberClassLoader`.